### PR TITLE
Added stronger typing using ::class for services

### PR DESCRIPTION
**Pull request by Koen1999**
@tomudding , I now only did this for the translator. But I think this is a direction we should take for all services.
What do you think?

### CODE REVIEW

File: module/Frontpage/src/Module.php [hunk 1]
```diff
@@ -48,8 +49,7 @@ public function getServiceConfig(): array
         return [
             'factories' => [
                 'frontpage_service_frontpage' => function (ContainerInterface $container) {
-                    $translator = $container->get('translator');
-                    $aclService = $container->get('decision_service_acl');
```
Comment (tomudding) [inline module/Frontpage/src/Module.php:52]: Should not be removed?

Comment (tomudding) [reply module/Frontpage/src/Module.php:52]: The `$aclService` I mean.

**Review by tomudding — COMMENTED** [review]
Yes, this will make working on the website a lot better in the future. I think it is a good idea to only do this with the `MvcTranslator` to see how it plays out (e.g., we might want to look into potential performance impacts - positive and negative).

File: module/Frontpage/src/Module.php (hunk 1 above)
Comment (Koen1999) [reply module/Frontpage/src/Module.php:52]: I'm not sure why this went wrong while rebasing...Anyway, fixed it

**Comment by tomudding** [comment]
I switched the base branch to remove the commits already on the target branch (to make it easier to do the merge).

**Pull request merged**